test/Java8andUp/testacc.policy

```diff
@@ -0,0 +1,26 @@
+//  Copyright (c) 2018, 2018 IBM Corp. and others
```

Comment:
You are really taking Peter Shipton's comment about its okay to use // for comments seriously here for this 20 line block of commented code.  Out of curiosity, are policy files limited to using that style of comment or can /* */ also be used?